src/Core/Framework/DataAbstractionLayer/Write/Command/ChangeSet.php

```diff
@@ -8,32 +8,25 @@
 #[Package('framework')]
 class ChangeSet extends Struct
 {
-    protected array $state = [];
-
     protected array $after = [];
 
-    protected bool $isDelete;
-
     public function __construct(
-        array $state,
+        protected array $state,
```

Comment:
```suggestion
        protected array $state = [],
```

should hopefully fix one of the BC checker issues